jenkins/scripts/bml_integration_test.sh

```diff
@@ -43,6 +43,7 @@ EPHEMERAL_TEST="${EPHEMERAL_TEST:-false}"
 BARE_METAL_LAB=true
 
 TEST_EXECUTER_IP="129.192.80.20"
+INTERNAL_SERVER_IP="192.168.1.3"
```

Comment:
I think we need to make sure that file description on the script usage in the beginning of this file is also up-to-date with the current changes (GitHub does not allow me to comment directly on that lines)

Reply:
Good catch. I will add the changes for fetching logs script as well

Reply:
Looks like logs collected properly now